### PR TITLE
move e2e testing to app-dev

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
This should take care of step 1 in #10419.

CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

**Comment by garyverhaegen-da** [comment]
Rendered version [here](https://garyverhaegen-da.github.io/daml-docs/).

File: docs/source/app-dev/bindings-ts/testing.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@
 Testing Your App
```
Comment (cocreature) [inline docs/source/app-dev/bindings-ts/testing.rst:4]: I think we should rephrase the title a bit. This is really not about general daml testing, it’s very specifically about e2e UI testing for web frontends. Maybe "E2e UI testing of Daml Applications"?

**Review by cocreature — APPROVED** [review]
thanks!

File: docs/source/app-dev/bindings-ts/testing.rst (hunk 1 above)
Comment (garyverhaegen-da) [reply docs/source/app-dev/bindings-ts/testing.rst:4]: I was assuming the context made it clear (this is under the "JavaScript Client Libraries" header), but happy to change if you don't think context is clear enough:

Screenshot 2021-08-04 at 14 58

Comment (garyverhaegen-da) [reply docs/source/app-dev/bindings-ts/testing.rst:4]: Maybe "Testing Your Web App"?

Comment (cocreature) [reply docs/source/app-dev/bindings-ts/testing.rst:4]: I like testing your web app :+1:

**Pull request merged**